packages/providers/file-system/src/config/schema.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2024 Mytra Control S.L. All rights reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be found in the LICENSE file
+ * or at https://opensource.org/licenses/MIT.
+ */
+
+import Joi from 'joi';
+
+export const schema = Joi.object({
```

Comment:
- this schema has more fields that the fields you need, check it using other providers

Reply:
Done.